python/lsst/sims/photUtils/SignalToNoise.py

```diff
@@ -7,22 +7,45 @@
           "calcM5", "calcSkyCountsPerPixelForM5", "calcGamma", "calcSNR_m5",
           "calcAstrometricError", "magErrorFromSNR", "calcMagError_m5", "calcMagError_sed"]
 
+def FWHMeff2FWHMgeom(FWHMeff):
+    """
+    @param [in] FWHMeff (the single-gaussian equivalent FWHM value, appropriate for calcNeff) in arcseconds
 
-def calcNeff(seeing, platescale):
+    @param [out] FWHMgeom (the geometric FWHM value, as measured from a typical PSF profile) in arcseconds
     """
-    Calculate the effective number of pixels in a double Gaussian PSF
+    FWHMgeom = 0.822*FWHMeff + 0.052
```

Comment:
Is there a reference where this expression comes from?  It would be helpful to include that, either in the docstring or in a comment, so that whoever is tasked with maintaining this code is not confused what is going on.